### PR TITLE
{math}[GCCcore/9.3.0] Gurobi v9.0.1 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/Gurobi/Gurobi-9.0.1-GCCcore-9.3.0-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+builddependencies = [
+    ('binutils', '2.34'),
+]
+
+dependencies = [
+    ('Python', '3.8.2'),
+]
+
+# remove bundled Python interpreter in favour of the dependency in EB
+postinstallcmds = ['rm %(installdir)s/bin/python3.7']
```
Comment (boegel) [inline easybuild/easyconfigs/g/Gurobi/Gurobi-9.0.1-GCCcore-9.3.0-Python-3.8.2.eb:30]: @lexming Maybe this should be a bit more generic?

```python
postinstallcmds = ['rm %(installdir)s/bin/python*']
```

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2635.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/dedc658d0a33f1720f7b876cf1fe5c0a for a full test report.

File: easybuild/easyconfigs/g/Gurobi/Gurobi-9.0.1-GCCcore-9.3.0-Python-3.8.2.eb (hunk 1 above)
Comment (lexming) [reply easybuild/easyconfigs/g/Gurobi/Gurobi-9.0.1-GCCcore-9.3.0-Python-3.8.2.eb]: fixed :+1:

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node381.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/8fcb90729d00f6820227874e4175c2c0 for a full test report.

**Comment by lexming** [comment]
@boegelbot yes :+1:

**Comment by lexming** [comment]
@boegel  yes :+1:

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3108.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/f282ddc3742e2cb38f465c9c2d84d7e5 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @lexming!

**Pull request merged**